unreleased/enhance-httpserversettings-healthcheck-extension.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: breaking
+component: healthcheckextension
+note: Remove deprecated `port` field from config.
```

Comment:
Two changelog entries?

Reply:
Needed because there are two types of changes in this PR- breaking(removing port field from config) and enhancement(use httpserversetting)

I can understand why we can split them across 2 PRs, but I believe this one was a small change to incorporate in one. Let me know if you think otherwise. Thanks

Reply:
A bit unconventional, but I don't see why we can't do it.